android/src/main/java/com/bitmovin/player/reactnative/services/MediaSessionPlaybackService.kt

```diff
@@ -0,0 +1,55 @@
+package com.bitmovin.player.reactnative.services
+
+import android.content.Intent
+import android.os.Binder
+import android.os.IBinder
+import com.bitmovin.player.api.Player
+import com.bitmovin.player.api.media.session.MediaSession
+import com.bitmovin.player.api.media.session.MediaSessionService
+
+class MediaSessionPlaybackService : MediaSessionService() {
+    inner class ServiceBinder : Binder() {
+        var player: Player?
+            get() = this@MediaSessionPlaybackService.player
+            set(value) {
+                disconnectSession()
+                this@MediaSessionPlaybackService.player = value
+                value?.let {
+                    createSession(it)
+                    connectSession()
+                }
```

Comment:
Nit: Would be a nice improvement to not destroy and recreate the media session in case the same player is set again